community/index.md

```diff
@@ -0,0 +1,51 @@
+# Community
+
+OpenSSL source is maintained by a [team of committers](committers.html).
+The overall project is run by the [OpenSSL Management Committee](omc.html).
+Technical decisions are made by the [OpenSSL Technical Committee](otc.html).
+We operate under a set of [project bylaws](/policies/omc-bylaws.html) and
+ask everyone to follow our [code of conduct](conduct.html).
+
+Here are some of the ways you can join the community and contribute;
+see the links on the right-hand side.
+The \"[getting started](getting-started.html)\" page has some ideas.
+We maintain several [mailing lists](mailinglists.html).
+Anyone can join, but you must be a member of a list to post to it.
+We have a [public wiki](https://wiki.openssl.org), and anyone can
+request an account and start adding content.
+We have a [team blog](/blog), where members of the development team
+will occasionally post.
+
+While we only distribute source, some members of the community make
+[binaries](binaries.html) available.
+
+Finally we'd also like to [thank](thanks.html) several groups for help
+with the project infrastructure over time.
+
+## Reporting Security Bugs {#securityreports}
```

Comment:
The `{#securityreports}` is being rendered and it shouldn't be.
`<a name="securityreports"></a>` is what we've used elsewhere to tag a place in a file.

Reply:
Sure, I can do that